tests/TestCase/Command/PluginAssetsCommandsTest.php

```diff
@@ -122,7 +121,7 @@ public function testSymlinkWhenVendorDirectoryExits()
         $this->exec('plugin assets symlink');
         $path = $this->wwwRoot . 'company' . DS . 'test_plugin_three';
         if (DS === '\\') {
-            $this->assertDirectoryExits($path);
+            $this->assertDirectoryExists($path);
```

Comment:
I am guessing this test never runs?

Reply:
Jep, sure looks like it then. Only relevant for windows from what I see.

Reply:
Should we remove it or mark it as window-specific to force it to run? Feels bad to leave a test that doesn't run.

Reply:
There are several more of these in this file.
Admad added this:

        $this->skipIf(
            DS === '\\',
            'Skip AssetsTask tests on windows to prevent side effects for UrlHelper tests on AppVeyor.'
        );

Thats why those are dead. Not sure what to do here.

Reply:
We could probably look into the windows issue as separate PR.